draft-ietf-rtcweb-jsep.xml

```diff
@@ -3826,23 +3824,38 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                         <t>For each specified payload type that is also
                         supported by the local implementation, establish
-                        a mapping between the payload type ID and the
-                        actual media format. [TODO - Justin to add more
-                        to explain mapping.] If any indicated payload
-                        type is unknown, it MUST be ignored.  [TODO:
-                        should fail on answers]</t>
+                        a mapping between the specified payload type ID and the
+                        actual media format, as described in
+                        <xref target="RFC3264" />, Section 6.1.
+                        Specifically, this means that the
+                        implementation determines the payload type ID to
+                        be used in outgoing RTP packets when sending each
+                        specified payload type. If any indicated payload
+                        type is unknown, it MUST be ignored.</t>
```

Comment:
This section is specifically about handling a remote description, so only mapping #2 is in scope here, right?